Formula/gmailctl.rb

```diff
@@ -0,0 +1,17 @@
+class Gmailctl < Formula
+  desc "Declarative configuration for Gmail filters"
+  homepage "https://github.com/mbrt/gmailctl"
+  url "https://github.com/mbrt/gmailctl/archive/v0.6.0.tar.gz"
+  sha256 "e188d5d18ac84ca86c94fb2b90d219ae3bea1e1ddee1966fae904c93bf27f233"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-ldflags", "-s -w -X main.version=#{version}",
```

Comment:
You should use `*std_go_args` here, which will make things a bit cleaner.
```
system "go", "build", "-ldflags", "-s -w -X main.version=#{version}",
       *std_go_args, "cmd/gmailctl/main.go"
```

For now, `*std_go_args` contains `"-trimpath", "-o", bin/name`.

Reply:
Nice, thanks for the suggestion.